ci-operator/step-registry/platform-external/ccm/conf/aws/platform-external-ccm-conf-aws-commands.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function echo_date() {
+  echo "$(date -u --rfc-3339=seconds) - $*"
+}
+
+test -f "${SHARED_DIR}/infra_resources.env" && source "${SHARED_DIR}/infra_resources.env"
+
+
+if [[ "${PLATFORM_EXTERNAL_CCM_ENABLED-}" != "yes" ]]; then
+  echo_date "Ignoring CCM Installation setup. PLATFORM_EXTERNAL_CCM_ENABLED!=yes [${PLATFORM_EXTERNAL_CCM_ENABLED}]"
+  exit 0
+fi
+
+# Build from: https://github.com/openshift/cloud-provider-aws/blob/master/Dockerfile.openshift
+CCM_IMAGE="quay.io/mrbraga/openshift-cloud-provider-aws:latest"
+CCM_NAMESPACE=openshift-cloud-controller-manager
+CCM_MANIFEST=ccm-00-deployment.yaml
+CCM_MANIFEST_PATH="${SHARED_DIR}"/${CCM_MANIFEST}
+
+echo_date "Creating CloudController Manager deployment"
+
+cat << EOF | envsubst > $CCM_MANIFEST_PATH
```

Comment:
Should we need to create the "base" deployment file, then in later steps patch with customizations for each provider?
Example steps:
- `platform-external-ccm-conf`: creates the deployment
- `platform-external-ccm-conf-aws`:  patch the deployment with specific config (like image, command, ports, etc)